tests/GraphBLAS-sharp.Tests/Program.fs

```diff
@@ -17,9 +17,9 @@ let allTests =
           Backend.RemoveDuplicates.tests
           Backend.Copy.tests
           Backend.Replicate.tests
-          Backend.EwiseAdd.tests
-          Backend.EwiseAdd.tests2
-          //Backend.EwiseAdd.tests3
+          Backend.Elementwise.tests
```

Comment:
Can you propose more user-friendly names than ```tests```, ```tests2```, ```tests3```. What the difference betwee these tests?